package.json

```diff
@@ -127,7 +127,7 @@
                     "group": "navigation@2"
                 },
                 {
-                    "command": "azureFunctions.deployZip",
+                    "command": "azureFunctions.deploy",
                     "when": "view == azureFunctionsExplorer",
                     "group": "navigation@3"
                 },
```

Comment:
nit: I feel like "Create Function App in Azure" should go above "Refresh" in the context menu